lib/src/reactive_builder.dart

```diff
@@ -22,13 +22,13 @@ class ReactiveBuilder<T> extends ReactiveBaseWidget<T> {
 
   @override
   Widget placeHolderBuild(BuildContext context) {
-    if (placeHolderBuilder != null) return placeHolderBuilder(context);
+    if (placeHolderBuilder != null) return placeHolderBuilder!(context);
     return super.placeHolderBuild(context);
   }
 
   @override
-  Widget errorBuild(BuildContext context, Object error) {
-    if (errorBuilder != null) return errorBuilder(context, error);
+  Widget errorBuild(BuildContext context, Object? error) {
```

Comment:
Why is error nullable here? Normally the ErrorBuilder should never be called if there is no error or do I miss something here?